bot/exts/moderation/alts.py

```diff
@@ -0,0 +1,174 @@
+import gettext
+
+import discord
+from discord.ext import commands
+from pydis_core.site_api import ResponseCodeError
+from pydis_core.utils.members import get_or_fetch_member
+
+from bot import constants
+from bot.bot import Bot
+from bot.converters import UnambiguousMemberOrUser
+from bot.log import get_logger
+from bot.pagination import LinePaginator
+from bot.utils.channel import is_mod_channel
+from bot.utils.time import discord_timestamp
+
+log = get_logger(__name__)
+
+
+class AlternateAccounts(commands.Cog):
+    """A cog used to track a user's alternative accounts across Discord."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    @staticmethod
+    def error_text_from_error(error: ResponseCodeError) -> str:
+        """Format the error into a user-facing message."""
+        if resp_json := error.response_json:
+            errors = ", ".join(
+                resp_json.get("non_field_errors", []) +
+                resp_json.get("source", []) +
+                resp_json.get("target", []) +
+                resp_json.get("detail", [])
+            )
+            if errors:
+                return gettext.ngettext("Error from site: ", "Errors from site: ", len(errors)) + errors
+            return str(error.response_json)
+        return error.response_text
+
+    async def alts_to_string(self, alts: list[dict[str: str | int | list[int]]]) -> list[str]:
+        """Convert a list of alts to a list of string representations."""
+        lines = []
+        guild = self.bot.get_guild(self.bot.guild_id)
+        for idx, alt in enumerate(alts):
```

Comment:
Might be better to start enumerating from `1` using the `start` kwarg here

Reply:
We use 0-based indexing around here!